packages/m2m-gateway/src/services/purposeService.ts

```diff
@@ -59,5 +78,25 @@ export function purposeServiceBuilder(clients: PagoPAInteropBeClients) {
         throwNotFoundError: true,
       });
     },
+    createPurpose: async (
+      purposeSeed: m2mGatewayApi.PurposeSeed,
+      { logger, headers }: WithLogger<M2MGatewayAppContext>
+    ): Promise<m2mGatewayApi.Purpose> => {
+      logger.info(`Creating purpose`);
```

Comment:
Add some info to this log, like the `eserviceId` and the `consumerId` for which the purpose is being created?